gostrftime.go

```diff
@@ -110,7 +110,8 @@ func strftime(b *bytes.Buffer, c rune, t time.Time) error {
 		b.WriteString(zone)
 	case 'z':
 		_, offset := t.Zone()
-		fmt.Fprintf(b, "%+05d", offset)
+		allMinutes := int(offset/60)
+		fmt.Fprintf(b, "%+02d%02d", int(allMinutes / 60), int(allMinutes % 60))
```

Comment:
Yeah, you're right. I'm not sure why i've pushed version with invalid
formatting as i'm having done it right locally. I'll fix this. Thx for so
fast response.

On Sat, May 23, 2015 at 9:00 PM, eli <notifications@github.com> wrote:

> In gostrftime.go
> <https://github.com/cactus/gostrftime/pull/2#discussion_r30947309>:
>
> > @@ -110,7 +110,8 @@ func strftime(b *bytes.Buffer, c rune, t time.Time) error {
> >  		b.WriteString(zone)
> >  	case 'z':
> >  		_, offset := t.Zone()
> > -		fmt.Fprintf(b, "%+05d", offset)
> > +		allMinutes := int(offset/60)
> > +		fmt.Fprintf(b, "%+02d%02d", int(allMinutes / 60), int(allMinutes % 60))
>
> Should be "%+03d%02d" so that the + is followed by two digits. As it is
> now you can end up with numbers like +159.
> Other than that, this looks ok. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cactus/gostrftime/pull/2/files#r30947309>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.